File: itt-frontend/src/influxdb.js
import { InfluxDB } from '@influxdata/influxdb-client';

const url = 'http://localhost:8086';
const token = 'my-token';
const org = 'my-org';
const bucket = 'my-bucket';

const client = new InfluxDB({ url, token });

export { client, org, bucket };

File: itt-backend/Influxdb.py
from influxdb_client import InfluxDBClient
from influxdb_client.client.write_api import SYNCHRONOUS

class InfluxDB:
    def __init__(self, url, token, org):
        self.client = InfluxDBClient(url=url, token=token, org=org);
        self.influxdbwriter = self.client.write_api(write_options=SYNCHRONOUS)
